fuzzing/private/java_utils.bzl

```diff
@@ -80,51 +81,65 @@ def _java_segment_index(path_segments):
 
     return root_index
 
-def _jazzer_fuzz_binary_script(ctx):
+def _jazzer_fuzz_binary_script(ctx, native_libs, driver):
     script = ctx.actions.declare_file(ctx.label.name)
 
-    script_template = """
-exec "{driver}" \
-    --agent_path="{agent}" \
-    --cp="{deploy_jar}" \
-    --jvm_args="-Djava.library.path={library_path}" \
+    # The script is split into two parts: The first is emitted as-is, the second
+    # is a template that is passed to format(). Without the split, curly braces
+    # in the first part would need to be escaped.
+    script_literal_part = """#!/bin/bash
+# LLVMFuzzerTestOneInput - OSS-Fuzz needs this string literal to appear
+# somewhere in the script so it is recognized as a fuzz target.
+
+# --- begin runfiles.bash initialization v2 ---
+# Copy-pasted from the Bazel Bash runfiles library v2.
```

Comment:
Could we reuse the library here, instead of copy-pasting it?

Reply:
That is unfortunately not possible, see https://github.com/bazelbuild/bazel/blob/e3c4f4369c3c37d14d95146b69e49386290a988f/tools/bash/runfiles/runfiles.bash#L50. We have a chicken-and-egg problem where we first need to find the library, which is itself a runfile, before we can use it to find runfiles.

Reply:
Oh, I see! Could you leave a comment in this respect? E.g. "Bazel-provided code snippet that should be copy-pasted as is at use sites."

Reply:
Sounds good!